agimus_controller_ros/agimus_controller_ros/simple_trajectory_publisher.py

```diff
@@ -134,15 +141,15 @@ def publish_mpc_input(self):
         ee_pose = self.pin_data.oMf[self.ee_frame_id]
         xyz_quatxyzw = pin.SE3ToXYZQUAT(ee_pose)
 
-        u = pin.computeGeneralizedGravity(self.pin_model, self.pin_data, self.q0)
+        u = pin.rnea(self.pin_model, self.pin_data, self.q, self.dq, self.ddq)
```

Comment:
Those changes (this one and the one below) might be out of scope. Feel free to keep them if they don't hurt.